syntax = "proto3";

package rj_GroupService;

message AddGroupRequest
{
    int32 userid = 1;
    int32 group_id = 2;
}

message CreateGroupRequest
{
    int32 userid = 1;
    bytes group_name = 2;
    bytes group_desc = 3;
}

message CreateGroupResponse
{
    int32 grouid = 1;
}

message QuitGroupRequest
{
    int32 userid = 1;
    int32 groupid = 2;
}

message GetGroupUsersRequest
{
    int32 group_id = 1;
}

message  UserInfo
{
    int32 id = 1;
    bytes name = 2;
}

message GetGroupUsersResponse
{
    bool is_success = 1;
    repeated UserInfo users = 2;
}
